src/OpenTelemetry.Exporter.Instana/OpenTelemetry.Exporter.Instana.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+	<GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+	<PackageId>OpenTelemetry.Exporter.Instana</PackageId>
+	<PackageTags>Instana Tracing APM</PackageTags>
+	<Copyright>Copyright 2019-</Copyright>
```

Comment:
Please cleanup this - you can copy most from any existing projects.

Reply:
Fixed